ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -124,4 +127,23 @@ public void shouldNotEnforceTopicExistanceWhileParsing() throws Exception {
 
   }
 
+  @Test
+  public void shouldCleanUpSchemaRegistry() throws Exception {
+    final List<QueryMetadata> queries
```

Comment:
Be nice to have each statement on its own line, plus `queries` is never used.

```java
ksqlEngine.buildMultipleQueries(
        "create table bar with (value_format = 'avro') as select * from test2;" +
        "create table foo as select * from test2;",
        Collections.emptyMap());
```

Reply:
We should still get rid of the `final List<QueryMetadata> queries = `